packages/flutter/lib/src/material/text_selection.dart

```diff
@@ -32,8 +34,9 @@ class _TextSelectionToolbar extends StatelessWidget {
     }
     items.add(new FlatButton(
       child: new Text('PASTE'),
-      onPressed: delegate.pasteBuffer != null ? _handlePaste : null)
-    );
+      // TODO(mpcomplete): should this be grayed-out if there is nothing to paste?
```

Comment:
https://flutter.io/style-guide/#avoid-checking-in-comments-that-ask-questions :-)